test/data/invalid-blank.csv

```diff
@@ -0,0 +1 @@
+
```

Comment:
Yeah @rclark, I ran the file sniffer on all files in https://github.com/mapnik/test-data/tree/master/csv and copied over any files that were initially problematic. At the moment, all valid files in the mapnik/test-data csv folder have been tested and are coming back positive.

We have two options:

1. **Depending on mapnik-test-data** means we could run the sniffer against all valid CSVs.
2. **Keeping copies** of just the tricky files in `/test/data` makes us consistent with the other file type tests in the sniffer.

Easy switch, but I'm partial to keeping things consistent (**option 2**).